magda-web-client/src/Components/Dataset/DatasetSuggestForm.scss

```diff
@@ -2,13 +2,17 @@
 
 .ask-dataset-form {
     padding: 10px;
-    border: solid;
-    border-color: $AU-color-foreground-focus;
-    background-color: $AU-colordark-foreground-action;
     overflow: -webkit-paged-y;
     position: relative;
     float: right;
     width: 800px;
+    background-color: rgb(248, 247, 247);
```

Comment:
Can we use $AU-colordark-foreground-action instead? it is rgb(241,247,238) which is very close to this color here and can help keep the theme consistent

Reply:
hi @chloeleichen  that color looks really bad. Please see below

![image](https://user-images.githubusercontent.com/20184809/41581838-3833c394-73e3-11e8-95c2-9f920c181191.png)
****

Reply:
yeah, i agree. Happy to use the new color 💃

Reply:
😸